packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -283,7 +339,7 @@ async function callDestinationEndpoint(
 ): Promise<AxiosResponse<DestinationJson | DestinationConfiguration>> {
   if (!uri.match(/[instance|subaccount]Destinations|v1\/destinations/)) {
     throw new Error(
-      `callDestinationEndpoint was called with illegal arrgument: ${uri}. URL must be destination(s) endpoint of destination service.`
+      `callDestinationEndpoint was called with illegal argument: ${uri}. URL must be destination(s) endpoint of destination service.`
```

Comment:
[req] I can see that this was here before, but this is not a good error message. Users do not have access to `callDestinationEndpoint` and most certainly did not call it directly. This error will never happen unless we make a major mistake in our code, but still, I think we can improve the message:
```suggestion
      `Could not call destination service with URL: "${uri}". URL must be an endpoint of the destination service.`
```